build.sbt

```diff
@@ -1,12 +1,13 @@
 import com.github.bigtoast.sbtthrift.ThriftPlugin
 import sbt.ExclusionRule
-import com.typesafe.sbt.SbtGit.{GitKeys => git}
+import com.typesafe.sbt.SbtGit.GitKeys._
 
-val specs2Ver = "3.6.4"
+val specs2Ver = "3.8.6"
```

Comment:
There is a bug in specs2 preventing us from upgrading past `3.8.6` (latest is `3.8.9`). I will endeavour to get a test-case for this and figure out a solution separately. For now, this at least allows us to upgrade `scalaz` to the latest version.